lib/exchange/sideshift/sideshift_exchange_provider.dart

```diff
@@ -150,6 +150,7 @@ class SideShiftExchangeProvider extends ExchangeProvider {
       refundAddress: settleAddress,
```

Comment:
why are we using the `settleAddress` as the `refundAddress`? we already have the refund address inside the request model.
I believe this is the reason why we are not using `settleAddress` as the `refundAddress` in `findTradeById`